src/langs/en.json

```diff
@@ -24,6 +24,7 @@
   "ITEM_CHATBOX_TITLE": "\"{{name}}\" Chat",
   "ITEM_TITLE_CREATED_AT": "Created: {{date}}",
   "ITEM_TITLE_UPDATED_AT": "Updated: {{date}}",
+  "HOME_PAGE_TITLE": "Player items",
```

Comment:
The title of the page should be aligned with the title that is shown on the page.